.github/workflows/pr_check_load_test-windows.yml

```diff
@@ -13,18 +13,22 @@ on:
       - main
 
 env:
-  LOAD_TEST_RESOURCE: "rbac-test"
+  LOAD_TEST_RESOURCE: "mohit-1"
   LOAD_TEST_RESOURCE_GROUP: "radhika-test"
   COMMIT_ID : ${{ github.sha }}
 
+permissions:
+  id-token: write # This is required for requesting the JWTcontents: read  # This is required for actions/checkout
```

Comment:
nit: Correct the comment. This - 
contents: read # This is required for actions/checkout

Should be in next line

Reply:
done